src/components/signMessage/confirmMessage.css

```diff
@@ -82,14 +85,14 @@
 }
 
 .result {
-  padding: 20px;
+  padding: 20px 180px;
   background-color: var(--color-form-bg);
 
   & textarea {
-    background: #333;
+    background: #06213b;
     border-radius: 5px;
     overflow: auto;
-    color: var(--color-white);
+    color: #9bf436;
```

Comment:
All colors should be defined in `src/components/app/variables.css` in case we want to do a night mode in the future